### PR TITLE
builders: Set imagePullPolicy to always (PROJQUAY-3507)

### PR DESCRIPTION
**Pull request by bcaton85**
Currently imagePullPolicy is set to IfNotPresent. This will ignore updates to the builder image.

### CODE REVIEW

File: oauth/services/rhsso.py [hunk 1]
```diff
@@ -34,7 +34,10 @@ def exchange_code_for_login(self, app_config, http_client, code, redirect_suffix
 
                 # 200 => Endpoint was hit successfully and user was found
                 # 400 => Endpoint was hit successfully but no user was found
-                if result.status_code == 200 and result.json().get("result", "") == "ERROR_EXPORT_CONTROL":
+                if (
```
Comment (syed) [inline oauth/services/rhsso.py:37]: can you rebase for this

File: buildman/manager/executor.py [hunk 2]
```diff
@@ -552,7 +552,7 @@ def _build_job_containers(self, token, build_uuid):
 
         container = {
             "name": "builder",
-            "imagePullPolicy": "IfNotPresent",
+            "imagePullPolicy": "Always",
```
Comment (syed) [inline buildman/manager/executor.py:555]: As @kleesc  suggested, can we make this a config value?

**Review by syed — CHANGES_REQUESTED** [review]

File: oauth/services/rhsso.py (hunk 1 above)
Comment (bcaton85) [reply oauth/services/rhsso.py:37]: it looks like i'm already on top of the latest commit - b21400b90d46ff1d6a989c2b3008769420bbd608

File: buildman/manager/executor.py (hunk 2 above)
Comment (bcaton85) [reply buildman/manager/executor.py]: yes, updated to be configurable.

**Review by kleesc — APPROVED** [review]
LGTM

**Review by syed — APPROVED** [review]

**Pull request merged**